lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -304,6 +304,15 @@ MayaUsdProxyShapeBase* getProxyShape(const Ufe::Path& path)
     return UsdStageMap::getInstance().proxyShapeNode(path);
 }
 
+SdfPath getPrimPath(const Ufe::Path& path)
```

Comment:
Not sure I like the function name, I think it is misleading. One could think it return the USD prim path corresponding to the given UFE path, but it actually return the root prim path of the corresponding proxy shape. Maybe the function should be named getProxyShapePrimPath?

Reply:
> Not sure I like the function name, I think it is misleading. One could think it return the USD prim path corresponding to the given UFE path, but it actually return the root prim path of the corresponding proxy shape. Maybe the function should be named getProxyShapePrimPath?

Makes sense

Reply:
ahah, I saw it too but chose not to mention it since you did not write it. You can leave it as is, as there is no time associated with prims anyway, so it is less confusing, even though I was tempted to suggest the change too.